src/custom/simplifions/components/SimplifionsSolutionDescription.vue

```diff
@@ -0,0 +1,288 @@
+      <span
+        class="fr-icon-loader-line fr-icon--rotate"
+        aria-hidden="true"
+      ></span>
+      Chargement des cas d'usages...
+    </div>
+    <div
+      v-else-if="relatedCasUsages.length > 0"
+      class="fr-grid-row fr-grid-row--gutters"
+    >
+      <div
+        v-for="casUsage in relatedCasUsages"
+        :key="casUsage.id"
+        class="fr-col-12 fr-col-md-6 fr-col-lg-4 fr-mb-3w"
+      >
+        <div class="fr-tile fr-tile--horizontal fr-enlarge-link">
+          <div class="fr-tile__body">
+            <div class="fr-tile__content">
+              <h3 class="fr-tile__title">
+                <a href="/cas-d-usages/{{casUsage.slug}}">
+                  {{ casUsage.name }}
+                </a>
+              </h3>
+              <p class="fr-tile__detail">
+                {{ casUsage.description || 'Aucune description disponible' }}
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <p v-else class="fr-text--sm">
+      Aucun cas d'usage ne référence cette solution pour le moment.
+    </p>
+
+    <h2 class="colored-title fr-h2 fr-mt-8w fr-mb-0">
+      Données et API utilisées
+    </h2>
+  </div>
+</template>
+
+<style scoped>
+h2.colored-title {
+  color: black;
+  background-color: rgb(167, 212, 205);
+  padding: 2px 4px;
+  display: inline-block;
+}
+h3 {
+  color: #616161;
+}
+.example-image {
+  display: inline-block;
+  box-shadow: 0 4px 10px rgba(0, 0, 0, 0.2);
+  padding: 8px;
+  border: 4px solid #000;
+}
+
+.example-image img {
+  display: block;
+  width: 100%;
+  height: auto;
+  border-radius: 4px;
+}
+
+.inline-tag {
+  display: inline-flex;
+  margin-right: 0.5em;
+  margin-left: 0.5em;
+}
+
+@keyframes rotate {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.fr-icon--rotate {
+  animation: rotate 1s linear infinite;
+}
+</style>
+
+<script setup lang="ts">
+import OrganizationLogo from '@/components/OrganizationLogo.vue'
+import TagComponent from '@/components/TagComponent.vue'
+import type { Topic } from '@/model/topic'
+import TopicsAPI from '@/services/api/resources/TopicsAPI'
+import { formatDate, fromMarkdown } from '@/utils'
+import { useTagsByRef } from '@/utils/tags'
+import { OrganizationNameWithCertificate } from '@datagouv/components'
+import { onMounted, ref } from 'vue'
+import { gristImageUrl } from './simplifions_utils'
+
+const props = defineProps<{
+  topic: Topic
+  pageKey: string
+}>()
+
+const topicRef = ref(props.topic)
+const tags = useTagsByRef(props.pageKey, topicRef)
+const tags_niveau_simplification = tags.value.filter(
+  (t: any) => t.type === 'types-de-simplification'
+)
+const tags_budget = tags.value.filter((t: any) => t.type === 'budget')
+
+const solution = (props.topic.extras as any)['simplifions-solutions'] as Record<
+  string,
+  any
+>
+
+// Reactive variables for cas d'usages
+const loading = ref(false)
```

Comment:
Utiliser `useLoading` ? Cf https://github.com/opendatateam/udata-front-kit/blob/a13eaef7ae5c644188fb409428fd62b74707bcc1/src/components/datasets/DatasetList.vue#L75
Le composant est décrit comme "full screen" mais il y a une prop `is-full-page` ?

Reply:
Bien vu pour `useLoading`, merci.

Je l'ai utilisé, par contre j'ai pas compris cette question : 

> Le composant est décrit comme "full screen" mais il y a une prop is-full-page ?